pkg/syncer/util.go

```diff
@@ -133,3 +134,28 @@ func getPVCKey(ctx context.Context, obj interface{}) (string, error) {
 	log.Infof("getPVCKey: PVC key %s", objKey)
 	return objKey, nil
 }
+
+// GetVolumeID returns volume handle if successfully found by the volumeMigrationService
+func GetVolumeID(ctx context.Context, volumePath string) (string, error) {
```

Comment:
Additional function `func GetVolumeID(ctx context.Context, volumePath string) (string, error)` is not required. I see there is no additioanl logic in this wrapper func.

Can we directly call `volumeMigrationService.GetVolumeID(ctx, volumePath)`, instead of wrapping into GetVolumeID? I guess volumeMigrationService instance is directly accessible all the places.

Reply:
Makes sense. Done